src/server/running-services/helpers/fetch/fetch-running-services.js

```diff
@@ -11,9 +11,12 @@ async function fetchRunningServices(environments) {
       { environments: Object.values(environments) },
       { arrayFormat: 'repeat', addQueryPrefix: true }
     )
-
-  const { json } = await fetcher(endpoint)
-  return json
+  try {
+    const { json } = await fetcher(endpoint)
+    return json
+  } catch (error) {
+    return []
```

Comment:
Also wondering if it would be better to move this catch up to where its being called? That might give us an option of rendering something in the UI to indicate its not happy?

Reply:
So the error will be logged lower down for all of these in the fetcher